docs/02_Preparing_Environment_for_a_VM_Cluster.md

```diff
@@ -294,19 +294,19 @@ sudo ip link add kubr0 type bridge
 sudo ip link set kubr0 up
 ```
 
-We choose **192.168.1.0/24** as the CIDR for the network, where 192.168.1.1 is the host machine address.
+We choose **192.168.3.0/24** as the CIDR for the network, where 192.168.1.1 is the host machine address.
```

Comment:
we need to update `192.168.1.1` to `192.168.3.1` here

Reply:
I've fixed it.